### ISSUE TITLE
Feature request: Top K accuracy score

### ISSUE THREAD

**Issue by vijay-jaisankar** [mindsdb/mindsdb_evaluator#15]
Based on the [Slack discussion](https://mindsdbcommunity.slack.com/archives/C037482KJ22/p1678898027695699), I would like to work on adding [Top K accuracy](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.top_k_accuracy_score.html#sklearn.metrics.top_k_accuracy_score) to this project.

If you feel like this idea is worthwhile, please assign me to the same.

The corresponding PR will include
- Exposing this metric from `scikit-learn`
- Tests for the same

**Comment by paxcema** [comment]
Awesome @vijay-jaisankar, let us know if you need any help during the implementation! Once the PR is pushed, we'll review and provide feedback ASAP.

**Comment by vijay-jaisankar** [comment]
Sure thing, thanks a lot!

**Comment by paxcema** [comment]
Closed via #16
